src/core/src/op/reverse_sequence.cpp

```diff
@@ -59,25 +55,14 @@ shared_ptr<Node> op::ReverseSequence::clone_with_new_inputs(const OutputVector&
 
 void op::ReverseSequence::set_batch_axis(int64_t batch_axis) {
     m_batch_axis = batch_axis;
-    const auto& data_pshape = get_input_partial_shape(0);
-    const auto& data_rank = data_pshape.rank();
-    m_normalized_batch_axis = ov::normalize_axis(this, m_batch_axis, data_rank);
 }
 
-void op::ReverseSequence::set_sequence_axis(int64_t sequence_axis) {
-    m_seq_axis = sequence_axis;
-    const auto& data_pshape = get_input_partial_shape(0);
-    const auto& data_rank = data_pshape.rank();
-    m_normalized_seq_axis = ov::normalize_axis(this, m_seq_axis, data_rank);
+size_t op::ReverseSequence::get_batch_axis() const {
+    const auto& data_rank = get_input_partial_shape(0).rank();
+    return static_cast<size_t>(ov::normalize_axis(this, m_batch_axis, data_rank));
```

Comment:
Is there a specific reason for moving axis normalization from setter to getter?

And similar comment to `set_sequence_axis`:
`get_input_partial_shape(0)` makes impossible to call `get_batch_axis` on default constructed op without inputs, 
can it be changed to perform normalization `if inputs.size() > 0`?

Reply:
Yes as `m_batch_axis` is required for shape inference and if it is possible to set without input this is why is moved to getter so you can only get when you got input set. 
Get this value without normalization when there is no inputs will give not correct value so exception is OK for me.
For not normalized value there is `get_origin_batch_axis`.